roles/pulp_common/tasks/repos.yml

```diff
@@ -146,7 +136,18 @@
         gpgcheck: "{{ pulp_pkg_repo_gpgcheck }}"
         gpgkey: "{{ __pulp_pkg_repo_gpgkey }}"
         enabled: true
-        include: /etc/yum/pulpcore.conf
+      changed_when: false
+
+    # Set module_hotfixes=1 so that dnf allows to override packages that are part
+    # of modular repositories (like CentOS-8 - AppStream).
+    # Use a lineinfile module yum_repository doesn't support setting the
+    # module_hotfixes parameter yet.
+    - name: Set module_hotfixes to true
+      lineinfile:
+        path: "/etc/yum.repos.d/{{ __pulp_pkg_repo_name }}.repo"
+        line: module_hotfixes=1
+      changed_when: false
```

Comment:
changed_when: false sounds wrong here

Reply:
if yum_repo/lineinfile produce changes all the time (thus failing molecules idempotency check), the easiest solution is just to use `copy` with a template.

Reply:
It is wrong, but I didn't know if changing the whole structure was ok. Will uppddate for a template. Thanks for review.